src/components/registration/steps.jsx

```diff
@@ -324,7 +324,7 @@ module.exports = {
         },
         getYearOptions: function () {
             return Array.apply(null, Array(100)).map(function (v, id) {
-                var year = 2016 - id;
+                var year = 2017 - id;
```

Comment:
Yep that JS solution works!  React tries not to get involved with "business logic" like this, and  only provides tools for building the UI.